### PR TITLE
[FEATURE] Adds debugbar key to doctrine collector.

### PR DESCRIPTION
**Pull request by zquintana**
This PR is for the proposal in https://github.com/laravel-doctrine/orm/issues/237. It adds config for the doctrine debugbar widget name. It was mentioned to use the `env`, but I added it to the configs because the use of `env` outside of configs is deprecated in laravel 5.2+ and it's use doesn't work when configs are cached.

The new config name is `debugbar_widget_key` and it defaults to the current value. Setting it in config will allow the eloquent debugbar to show with the doctrine debugbar.

### CODE REVIEW

File: src/Loggers/Debugbar/DoctrineCollector.php [hunk 1]
```diff
@@ -13,6 +14,23 @@ class DoctrineCollector extends DebugbarDoctrineCollector
     protected $debugStack;
 
     /**
+     * @var string
+     */
+    protected $name;
```
Comment (guiwoda) [inline src/Loggers/Debugbar/DoctrineCollector.php]: I'd stay consistent with the `key` keyword here as well.

**Review by guiwoda — CHANGES_REQUESTED** [review]

**Comment by guiwoda** [comment]
Please fix the tests as well

File: config/doctrine.php [hunk 2]
```diff
@@ -143,6 +143,8 @@
     |--------------------------------------------------------------------------
     */
     'logger'                     => env('DOCTRINE_LOGGER', false),
+    'debugbar_widget_key'        => env('DOCTRINE_DEBUGBAR_KEY', 'queries'),
```
Comment (patrickbrouwers) [inline config/doctrine.php]: Don't think it's really needed to have a env variable for this. You would change this setting for your project, not per environment.

File: src/Loggers/Debugbar/DoctrineCollector.php [hunk 3]
```diff
@@ -13,6 +14,23 @@ class DoctrineCollector extends DebugbarDoctrineCollector
...
+
+
+    /**
+     * DoctrineCollector constructor.
```
Comment (patrickbrouwers) [inline src/Loggers/Debugbar/DoctrineCollector.php]: Ommit description

File: src/Loggers/Debugbar/DoctrineCollector.php (hunk 1 above)
Comment (patrickbrouwers) [reply src/Loggers/Debugbar/DoctrineCollector.php]: I would prefer `widgetName` as that would be more explicit.

File: config/doctrine.php (hunk 2 above)
Comment (patrickbrouwers) [reply config/doctrine.php]: Also think this should be `debugbar_widget_name`.

**Review by patrickbrouwers — CHANGES_REQUESTED** [review]
Thanks for your PR, looking good; just a few small remarks. Also please fix the failing tests.

File: config/doctrine.php (hunk 2 above)
Comment (zquintana) [reply config/doctrine.php]: Changed it to `debugbar_widget_name` in latest commit.

File: src/Loggers/Debugbar/DoctrineCollector.php (hunk 3 above)
Comment (zquintana) [reply src/Loggers/Debugbar/DoctrineCollector.php]: Removed the description too.

**Review by patrickbrouwers — CHANGES_REQUESTED** [review]
Please fix tests + code-style issue. (See merge checks)

**Comment by patrickbrouwers** [comment]
Replaced

**Pull request closed without merging**